doc/conseil.sql

```diff
@@ -308,6 +308,29 @@ CREATE TABLE public.rolls (
 );
 
 
+CREATE TABLE public.baking_rights (
+    block_hash character varying NOT NULL,
+    level integer NOT NULL,
+    delegate character varying NOT NULL,
+    priority integer NOT NULL,
+    estimated_time timestamp without time zone NOT NULL,
+    PRIMARY KEY(level, delegate)
+);
+
+CREATE INDEX baking_rights_level_idx ON public.baking_rights (level);
+
+CREATE TABLE public.endorsing_rights (
+    block_hash character varying NOT NULL,
+    level integer NOT NULL,
+    delegate character varying NOT NULL,
+    slots character varying NOT NULL,
```

Comment:
Isn't this supposed to be a single "slot", from the ticket description?
Each slot his own row

Reply:
you are right, I misunderstood it, I'm going to change that, good catch

Reply:
changed